src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -104,9 +116,9 @@ export class EditConcept extends Component {
     event.preventDefault();
   }
 
-  removeNewName(event, uuid) {
+  removeNewName(event, UUID) {
```

Comment:
What is the reason behind changing this to capital UUID?

Reply:
It was causing conflicts, there is already a dependency with the same name that I had imported. Therefore, `uuid` was already declared in the upper scope.

Reply:
Was that the only solution? Have you come across any convention that uses capital letters for variable names?

Reply:
@dkayiwa, that was not the only solution, I will just rename the variable to something better so it does not look bad.

Reply:
@dkayiwa, I have made changes to the variable name from `UUID` to `universalUniqueIdentifier`.

Reply:
Which universal unique identifier is it?